config.lua

```diff
@@ -129,3 +129,10 @@ Config.CharacterDataTables = {
     ['player_outfits'] = 'citizenid',
     ['player_vehicles'] = 'citizenid',
 } -- Rows to be deleted when the character is deleted
+
+---@type { name: string, amount: integer, metadata: table | fun(source: number): table}
+Config.StarterItems = { -- Character starting items
+    { name = 'phone', amount = 1 },
+    { name = 'id_card', amount = 1, metadata = function(source) if not lib.checkDependency('um-idcard', '1.2.8') then return else exports['um-idcard']:GetMetaLicense(source, {'id_card'}) end end },
```

Comment:
```suggestion
    { name = 'id_card', amount = 1, metadata = function(source) if not lib.checkDependency('um-idcard', '1.2.8', true) then error() end exports['um-idcard']:GetMetaLicense(source, {'id_card'}) end end },
```
Prints and errors the dependency. We want to fail loud here.